tests/framework/jailer.py

```diff
@@ -162,6 +162,27 @@ def jailed_path(self, file_path, create=False, create_jail=False):
             utils.run_cmd(cmd)
         return jailed_p
 
+    def copy_into_root(self, file_path, create_jail=False):
+        """Copy a file in the jail root, owned by uid:gid.
+
+        Copies a file in the jail root, creating the folder path if
+        not existent and changes their owner to uid:gid
+        """
+        global_path = os.path.join(
+            self.chroot_path(), file_path.strip(os.path.sep))
+        if create_jail:
+            os.makedirs(self.chroot_path(), exist_ok=True)
+
+        os.makedirs(os.path.dirname(global_path), exist_ok=True)
+
+        cmd = 'cp {} {}'.format(
+            file_path, global_path)
+        utils.run_cmd(cmd)
```

Comment:
No need to spawn a new process for a copy. You can use [`shutil.copy`](https://docs.python.org/3/library/shutil.html).

Reply:
done